erpnext/assets/report/fixed_asset_register/fixed_asset_register.py

```diff
@@ -138,19 +140,22 @@ def get_data(filters):
 				"location": asset.location,
 				"asset_category": asset.asset_category,
 				"purchase_date": asset.purchase_date,
-				"current_value": current_value_map.get(asset.name)
+				"asset_value": asset_value
 			}
 			data.append(row)
 
 	return data
 
-def get_finance_book_value_map(finance_book=''):
+def get_finance_book_value_map(finance_book='', date=today()):
```

Comment:
Don't write function in argument declaration, it creates problem (can't remember exactly what, but faced a problem earlier). Instead initialise inside the function like `if not date: date = today()`.

Reply:
added changes